husky_bringup/launch/bumblebee/bumblebee.launch

```diff
@@ -0,0 +1,38 @@
+<launch>
```

Comment:
I agree.  Should use environment variables since the serial which change per robot and other settings might need to be changed.  Check out the Jackal: https://github.com/jackal/jackal_robot/blob/indigo-devel/jackal_bringup/launch/accessories.launch#L18